src/ImageSharp/Formats/Jpeg/PdfJsPort/PdfJsJpegDecoder.cs

```diff
@@ -27,5 +27,16 @@ public Image<TPixel> Decode<TPixel>(Configuration configuration, Stream stream)
                 return decoder.Decode<TPixel>(stream);
             }
         }
+
+        /// <inheritdoc/>
+        public IImageInfo Identify(Configuration configuration, Stream stream)
```

Comment:
We need tests for pdfjs `Identify()`. (The test case is called `JpegDecoderTests.DetectPixelSize()` for the standard path now.)

Reply:
Agreed. Will get on that.